packages/token-list/types/index.d.ts

```diff
@@ -0,0 +1,28 @@
+import { MemoizedFunction } from 'lodash'
+
+export type ChainId = 1 | 137 | 43114
+type DevelopmentChainId = 1337 | 31337
```

Comment:
Hemi's Chain Id are missing here ☝🏽 

```suggestion
export type ChainId = 1 | 137 | 43114 | 43_111
type DevelopmentChainId = 1337 | 31337 | 743_111
```

Reply:
Added here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97
I suppose you didn't meant to add those `_` right?